OJP/OJP_Trips.xsd

```diff
@@ -106,11 +106,14 @@
 					<xs:documentation>The maximum number of interchanges the user will accept per trip.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="OptimisationMethod" type="OptimisationMethodEnumeration" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>the types of algorithm that can be used for planning a journey (fastest, least walking, etc)</xs:documentation>
-				</xs:annotation>
-			</xs:element>
+			<xs:choice>
+				<xs:element name="OptimisationMethod" type="OptimisationMethodEnumeration"/>
+				<xs:element name="OptimisationMethods" type="OptimisationMethodList" minOccurs="0">
```

Comment:
The possibility to pass several OptimisationMethods is a major change in my opinion.
The annotations of the other new OptimisationMethods (e. g. https://github.com/VDVde/OJP/pull/223/files) state the following:
"If multiple optimisation methods are needed, then several requests should be performed in parallel."
I guess we should discuss the expected impact on the basis of some use cases.